### DISCUSSION TITLE
freenom 已经完全挂掉了么？ 全部域名 pending

### DISCUSSION THREAD

**Discussion by appotry** [luolongfei/freenom#253] [Q&A] [answered]
所有域名active 变为 pending
续期也全部不可用了
There are no domains elligible for renewal in your account

**Comment by HelloAIXIAOJI** [comment]
要给官方发邮件

**Comment by HelloAIXIAOJI** [comment, marked as answer]
他们最近炸了，要发邮件才恢复

**Reply by appotry** [reply]
怎么发邮件？发到那个地址，有说明么

**Reply by HelloAIXIAOJI** [reply]
主题：Help me, dear Freenom team, to solve the domain name problem. Please! ! !
发送给：info@freenom.com
正文：
Dear Freenom Team,

I hope this message finds you well. I am reaching out to address an issue I have recently found with my Freenom domain names, which appear to have been deactivated for reasons unbeknownst to me.

Please let me assure you of my adherence to all Freenom guidelines and regulation, my use, and administration of my domains have always been in strict compliance with Freenom's Terms of Service.

My association with Freenom is valuable, as its services form the backbone of my business operations and website management. Therefore, the urgency of rectifying this situation cannot be overstated.

In the interests of expediting the resolution, I am hereby furnishing you with the details of my domain names and registration email.

Registration email: {你的Freenom绑定的邮箱}

I am fully prepared to cooperate with any verification process and can provide additional information or documents should they be required.

Thank you in advance for your kind assistance and understanding in this matter. I am hopeful that the full functionality of my domains can be swiftly reinstated.

Wishing you a productive day ahead.

Best Regards,
{你的名字}

**Reply by HelloAIXIAOJI** [reply]
我的就是这样恢复的

**Reply by HelloAIXIAOJI** [reply]
好了会自动延长9年

**Reply by HelloAIXIAOJI** [reply]
但有些域名不能起死回生

**Reply by appotry** [reply]
非常感谢，我试一下
